sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import tarfile
+import tempfile
+import os
+import uuid
+
+class ContainerBuilder(object):
+  """
+  ContainerBuilder helps build a container image
+  """
+  def __init__(self, gcs_staging):
+    if not self._check_gcs_path(gcs_staging):
+      raise Exception('ContainerBuilder __init__ failure.')
+    self._gcs_staging = gcs_staging
+
+  def _check_gcs_path(self, gcs_path):
+    """ _check_gcs_path check both the path validity and write permissions """
```

Comment:
This check is not part of the method. Do you want to correct the comment or adding a TODO to cover it.

Reply:
SG.

Reply:
Since the gcs_helper will throw meaningful error messages, thus correcting the comments here.